templates/content.html

```diff
@@ -3,6 +3,6 @@
 
 {% block content %}
     {% placeholder content or %}
-        <p>Hello world! This is HTML5 Boilerplate.</p>
+        <p>This page has no content yet, switch to structure mode and start adding content.</p>
```

Comment:
@evildmp we are not necessarily in edit mode yet. This might confuse the user if he doesnt see the "structure" mode button?

Adding an image might be a possibility. What kind of image would you add?